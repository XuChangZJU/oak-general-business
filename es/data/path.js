const paths = [
    {
        id: 'msg-user',
        sourceEntity: 'user',
        destEntity: 'message',
        value: 'user',
        recursive: false,
    },
    {
        id: 'mobile-user',
        sourceEntity: 'user',
        destEntity: 'mobile',
        value: 'user',
        recursive: false,
    },
    {
        id: 'token-user',
        sourceEntity: 'user',
        destEntity: 'token',
        value: 'user',
        recursive: false,
    },
    {
        id: 'wchL-user',
        sourceEntity: 'user',
        destEntity: 'wechatLogin',
        value: 'user',
        recursive: false,
    },
    {
        id: 'wchU-user',
        sourceEntity: 'user',
        destEntity: 'wechatUser',
        value: 'user',
        recursive: false,
    },
    {
        id: 'pwdCT-user',
        sourceEntity: 'user',
        destEntity: 'passwordChangeTemp',
        value: 'user',
        recursive: false,
    },
    {
        id: 'uec-user',
        sourceEntity: 'user',
        destEntity: 'userEntityClaim',
        value: 'user',
        recursive: false,
    },
];
export default paths;
