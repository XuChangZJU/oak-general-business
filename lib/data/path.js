"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const paths = [
    {
        id: 'msg-user',
        sourceEntity: 'user',
        destEntity: 'message',
        value: 'user',
        recursive: false,
    },
    {
        id: 'mobile-user',
        sourceEntity: 'user',
        destEntity: 'mobile',
        value: 'user',
        recursive: false,
    },
    {
        id: 'token-user',
        sourceEntity: 'user',
        destEntity: 'token',
        value: 'user',
        recursive: false,
    },
    {
        id: 'wchL-user',
        sourceEntity: 'user',
        destEntity: 'wechatLogin',
        value: 'user',
        recursive: false,
    },
    {
        id: 'wchU-user',
        sourceEntity: 'user',
        destEntity: 'wechatUser',
        value: 'user',
        recursive: false,
    },
    {
        id: 'pwdCT-user',
        sourceEntity: 'user',
        destEntity: 'passwordChangeTemp',
        value: 'user',
        recursive: false,
    },
    {
        id: 'uec-user',
        sourceEntity: 'user',
        destEntity: 'userEntityClaim',
        value: 'user',
        recursive: false,
    },
];
exports.default = paths;
