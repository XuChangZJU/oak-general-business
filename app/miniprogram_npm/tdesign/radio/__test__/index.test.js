import path from 'path';
import simulate from 'miniprogram-simulate';

describe('radio', () => {
  let id;
  beforeAll(() => {
    id = simulate.load(path.resolve(__dirname, '../radio'), { less: true });
  });
  test('comp should render correctly', async () => {
    const container = simulate.render(id);
    container.attach(document.createElement('parent-wrapper'));
    expect(container.toJSON()).toMatchSnapshot();
  });
  test('comp should trigger change events', async () => {
    const container = simulate.render(id);
    const that = container.instance;
    that.triggerEvent = jest.fn();
    that.onChange();
    expect(that.triggerEvent).toHaveBeenCalled();
  });
});
