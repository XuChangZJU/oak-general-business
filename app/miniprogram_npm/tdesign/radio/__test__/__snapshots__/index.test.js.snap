// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`radio comp should render correctly 1`] = `
Object {
  "attrs": Array [],
  "children": Array [
    Object {
      "attrs": Array [
        Object {
          "name": "class",
          "value": "t-radio ",
        },
      ],
      "children": Array [
        Object {
          "attrs": Array [],
          "children": Array [
            Object {
              "attrs": Array [
                Object {
                  "name": "class",
                  "value": "t-cell t-cell--bordered",
                },
              ],
              "children": Array [
                Object {
                  "attrs": Array [
                    Object {
                      "name": "class",
                      "value": "t-radio__button ",
                    },
                    Object {
                      "name": "slot",
                      "value": "left-icon",
                    },
                  ],
                  "children": Array [],
                  "event": Object {},
                  "tagName": "wx-view",
                },
                Object {
                  "attrs": Array [
                    Object {
                      "name": "class",
                      "value": "t-cell__title",
                    },
                  ],
                  "children": Array [],
                  "event": Object {},
                  "tagName": "wx-view",
                },
                Object {
                  "attrs": Array [
                    Object {
                      "name": "class",
                      "value": "t-cell__value",
                    },
                  ],
                  "children": Array [],
                  "event": Object {},
                  "tagName": "wx-view",
                },
              ],
              "event": Object {
                "tap": Object {
                  "handler": "onTap",
                  "id": 2,
                  "isCapture": false,
                  "isCatch": false,
                  "isMutated": false,
                  "name": "tap",
                },
              },
              "tagName": "wx-view",
            },
          ],
          "event": Object {
            "click": Object {
              "handler": "onChange",
              "id": 1,
              "isCapture": false,
              "isCatch": false,
              "isMutated": false,
              "name": "click",
            },
          },
          "tagName": "t-cell",
        },
      ],
      "event": Object {},
      "tagName": "wx-view",
    },
  ],
  "event": Object {},
  "tagName": "main",
}
`;
